docs/subnets/how-to-transfer-funds.md

```diff
@@ -0,0 +1,332 @@
+|        | index 1 |                         | P-avax1drppshkst2ccygyq37m2z9e3ex2jhkd2txcm5r |       0 | Mainnet |
++        +---------+                         +-----------------------------------------------+---------+---------+
+|        | index 2 |                         | P-avax10an3cucdfqru984pnvv6y0rspvvclz63e523m0 |     4.5 | Mainnet |
++        +---------+                         +-----------------------------------------------+---------+---------+
+|        | index 3 |                         | P-avax1yfpm7v5y5rej2nu7t2r0ffgrlpfq36je0rc5k6 |       0 | Mainnet |
++        +---------+                         +-----------------------------------------------+---------+---------+
+|        | index 4 |                         | P-avax17nqvwcqsa8ddgeww8gzmfe932pz2syaj2vyd89 |       0 | Mainnet |
++        +---------+                         +-----------------------------------------------+---------+---------+
+|        | index 5 |                         | P-avax1jzvnd05vsfksrtatm2e3rzu6eux9a287493yf8 |       0 | Mainnet |
++--------+---------+-------------------------+-----------------------------------------------+---------+---------+
+```
+
+The address `P-avax10an3cucdfqru984pnvv6y0rspvvclz63e523m0` has 4.5 AVAX and
+is associated to index 2 of ledger A.
+
+### Determine Receiver Address Index
+
+In this case the user wants to use index 0, the one CLI by default expects to contain funds.
+
+For the transfer command, it is also needed to know the target p-chain address. Do the following to
+obtain it:
+
+With the ledger B connected and the avalache app running, execute:
+
+```bash
+avalanche key list --mainnet --ledger 0
+```
+
+```text
++--------+---------+-------------------------+-----------------------------------------------+---------+---------+
+|  KIND  |  NAME   |          CHAIN          |                    ADDRESS                    | BALANCE | NETWORK |
++--------+---------+-------------------------+-----------------------------------------------+---------+---------+
+| ledger | index 0 | P-Chain (Bech32 format) | P-avax1r4aceznjkz8ch4pmpqrmkq4f3sl952mdrdt6xm |       0 | Mainnet |
++--------+---------+-------------------------+-----------------------------------------------+---------+---------+
+```
+
+Target address to be used is `P-avax1r4aceznjkz8ch4pmpqrmkq4f3sl952mdrdt6xm`, containing 0 funds.
+
+### Execute the Sending Part of the Transfer Operation
+
+A P-Chain to P-chain transfer is a two-part operation. There is no need for the two parts to be executed
+on the same machine, only for them to have some common params. For each part, the appropriate ledger
+(either source or target) must be connected to the machine executing it.
+
+First part starts the process, and moves the money out of the
+source account into a X-Chain account owner by the receptor. It needs to be signed by the sending ledger.
+
+The amount to be specified as input to the command is the amount to be received on the target address.
```

Comment:
Enter the amount of AVAX to send to the recipient. This amount does not include fees.